src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -143,6 +143,19 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
                         value: '$value'
                     }
                 );
+            } catch (e) {
+                return e.message;
+            }
+            """.trimMargin())
+        return when (result) {
+            is String -> false
+            else -> true
+        }
+    }
+
+    fun sendPresence(driver: RemoteWebDriver): Boolean {
```

Comment:
What does this do?

Reply:
this makes a call in the web client to force the sending of its presence, so that we can propagate out the fields we added (session id, mode, etc.)